### ISSUE TITLE
gtag - does not fire

### ISSUE THREAD

**Issue by ChromaticMind** [codediodeio/angular-gtag#1]
Hi there,

I have followed your readme to the letter but for some reason nothing happens. 

But - when I include the script file link ```<script async src="https://www.googletagmanager.com/gtag/js?id=<tag>"></script> ``` in index.html's <head> tag it fires both the hard-coded one and your service at the same time.

Without the link it fires neither.

If I check within the Google Tag Assistant extension (Chrome), when the above link is present, it shows 2 identical tags but without the link it shows no tags. 

Not sure what more I can do there so I hope you can shed some light on this.

Many thanks!

**Comment by tomaszzmuda** [comment]
I have to add the script file link too.

**Comment by codediodeio** [comment]
Does your config script contain this line? The send_page_view false is essential for preventing multiple fires. 

```
 gtag('config', 'UA-YOUR_TRACKING_ID', { 'send_page_view': false });
```

**Comment by tomaszzmuda** [comment]
In index.html I have following lines:
`  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-XXXXXXXX"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());
    gtag('config', 'UA-XXXXXXXX', { 'send_page_view': false });
  </script>`

It works. But only with import script in first line.
I have import in app.module with trackPageviews = true.

It's not a problem but at I thought the angular-gtag doesn't work for some reason. 
Updating readme with line of including js file should be fine.

**Comment by codediodeio** [comment]
Yes, it should be possible, but gtag specifically asks for it as the first item in the head. If you have a good solution, feel free to send a pull request, otherwise I will try to implement soon.

**Comment by vertigeaux** [comment]
Works fine for me in dev mode but won't work when I build with --prod

**Comment by codediodeio** [comment]
@vertigeaux to you get errors or just no analytics in prod?

**Comment by ChromaticMind** [comment]
@codediodeio yes I have ```gtag('config', 'UA-YOUR_TRACKING_ID', { 'send_page_view': false });``` in my index.html file along with ```import { GtagModule } from 'angular-gtag';

@NgModule({
  imports: [
    GtagModule.forRoot({ trackingId: 'UA-YOUR_TRACKING_ID', trackPageviews: true })
  ]
})``` in my app.module.ts.

If I remove the ```<script async src="https://www.googletagmanager.com/gtag/js?id=UA-XXXXXXXX"></script>``` line from index.html then ```this.gtag.pageview();``` never fires but when I do include it, it fires it twice.

Google Tag Assistant also shows 2 entries of the same tracking ID when the script src is included and 0 entries when it is not.

What am I doing wrong here or is there a bug in this package?

**Comment by vertigeaux** [comment]
@codediodeio No errors, I just get no analytics. 

Everything works fine in dev mode both locally and on the live site. I thought maybe the gtag package was getting shaken out of the prod build, so I tried it both with trackPageViews: true and gtag in my AppComponent constructor, and also I tried with trackPageViews: false and firing gtag.pageview() on router events. I get analytics either way in dev mode, but no analytics either way when I build with --prod.

I'm not experienced enough to know what to try next.

**Comment by codediodeio** [comment]
Yep, this is a bug. `ng build --prod` is stripping out the config. This will be fixed today.

**Comment by codediodeio** [comment]
Fixed in in release 1.0.3

Also, I added a `debug` option to the config that will console log all events to help with issues like this in the future.

**Comment by wafflesrg00d** [comment]
I seem to be running into the --prod issue, as well. Dev, locally, works fine, but no go in prod. Was this definitely fixed?
